bin/stop-application.sh

```diff
@@ -16,8 +16,8 @@
 # limitations under the License.
 #
 
-# shellcheck disable=SC2071
-if [[ $# < 1 ]] ; then
+
```

Comment:
ditto

Reply:
Sorry, I didn't see your reply, I'll fix it this weekend

Reply:
@bowenliang123 fix it,please check